src/ceph-daemon/ceph-daemon

```diff
@@ -1308,6 +1297,7 @@ def command_ceph_volume():
     if args.fsid:
         make_log_dir(args.fsid)
 
+    (uid, gid) = extract_uid_gid()
```

Comment:
I think these temp files can be root:root and we can skip extracting the uid/gid from the container

Reply:
I'm not sure about all `ceph-volume` command, but the basic ones seem aok when using the effective uid/gid ..